optimum/intel/openvino/modeling_text2speech.py

```diff
@@ -200,7 +199,7 @@ def __init__(
         **kwargs,
     ):
         self.config = config
-        self.use_cache = kwargs.get("use_cache", True)
+        self.use_cache = model_has_state(decoder)
```

Comment:
to stay consistent with other seq2seq models  https://github.com/huggingface/optimum-intel/blob/bdeea95b61a80b9b9217fb730b3609657b55d208/optimum/intel/openvino/modeling_seq2seq.py#L332-L333